community/consistency-check-legacy/src/main/java/org/neo4j/legacy/consistency/checking/full/FullCheck.java

```diff
@@ -161,7 +163,7 @@ static DiffRecordAccess recordAccess( StoreAccess store )
         T[] records = (T[]) Array.newInstance( type, (int) store.getHighId() );
         for ( int i = 0; i < records.length; i++ )
         {
-            records[i] = store.forceGetRecord( i );
+            records[i] = store.getRecord( i, store.newRecord(), FORCE );
```

Comment:
It's confusing for mocking also, which one should I mock (depends on which the code happens to call). This happened a lot during development of this change. You should care about when you allocate that record, if there's a method that creates it internally that for laziness reasons people are going to call that one. I'm moving the static methods from CommonAbstractStore to RecordStore, which can be used, although slightly harder to "find" but still can be used.